src/tests/components/Login.test.tsx

```diff
@@ -14,51 +15,105 @@ const renderComponent: ReactElement = (
     </ServiceProvider>
 )
 
-test('Login page render', () => {
-    render(renderComponent)
-    const loginPrimaryText = screen.getByText('Rucio Login')
-    expect(loginPrimaryText).toBeInTheDocument()
+const multi_vo = env('multi_vo_enabled')?.toLowerCase() === 'true'
 
-    const loginSecondaryText = screen.getByText('Welcome to Rucio!')
-    expect(loginSecondaryText).toBeInTheDocument()
+const itif = (condition: boolean) => (condition ? it : it.skip)
 
-    const x509PassAuthText = screen.getByText('x509 Certificate')
-    expect(x509PassAuthText).toBeInTheDocument()
+describe('Login page render', () => {
+    test('Titles', () => {
+        render(renderComponent)
+        const loginPrimaryText = screen.getByText('Rucio Login')
+        expect(loginPrimaryText).toBeInTheDocument()
 
-    const userPassAuthText = screen.getByText('Username / Password')
-    expect(userPassAuthText).toBeInTheDocument()
-})
+        const loginSecondaryText = screen.getByText('Welcome to Rucio!')
+        expect(loginSecondaryText).toBeInTheDocument()
+    })
+
+    itif(multi_vo)('Multi-vo Tabs Enabled', () => {
+        render(renderComponent)
+        const voTab1 = screen.getByTestId('Tab0')
```

Comment:
Instead of `getByTestId`, we could check if the tab with the name `ATLAS` exists and is the first tab in the list:
```
const atlasTab = screen.getByText('ATLAS')
expect(atlasTab.getAttribute('id')).toEqual('0')
```
Or we could check if all of the 'Tab' elements share the common (grand)parent.